apps/tx-builder/src/hooks/useServices/interfaceRepository.ts

```diff
@@ -76,7 +75,7 @@ class InterfaceRepository {
       .map((m: any) => {
         return {
           inputs: m.inputs || [],
-          name: m.name || (m.type === 'fallback' ? 'fallback' : 'receive'),
+          name: m.name || (m.type === 'fallback' ? 'fallback' : m.type === 'constructor' ? 'constructor' : 'receive'),
```

Comment:
Maybe we should first filter by known types. Currently we filter by "unwanted" types (e.g. event). If now in a future solidity version a new type is added we will run into a similar issue. Also the contructor should never be rendered to the user.